ci/cscs-ci.yml

```diff
@@ -156,7 +154,7 @@ build_py38_image_x86_64:
   variables:
     CRAY_CUDA_MPS: 1
     SLURM_JOB_NUM_NODES: 1
-    SLURM_TIMELIMIT: 15
+    SLURM_TIMELIMIT: 30
```

Comment:
The Slurm timeout should be changed back to 15 minutes, once the vCluster is stable.